README.md

```diff
@@ -2,130 +2,52 @@
 
 ## Contents
 - [Usage](#usage)
-  - [Using with Jekyll](#using-with-jekyll)
-  - [Using the plugin](#using-the-plugin)
+- [User Showcase](#user-showcase)
 - [Contributing](#contributing)
 
 ## Usage
-There are two ways to quickly get started using this theme. First, answer this question:
+Primer Spec is a Jekyll theme, which means you can start using this theme with [GitHub Pages](https://pages.github.com) right away!
 
-*Do you want to store your long webpage as MarkDown or as HTML?*
+1. Add your Markdown/HTML files to the Jekyll site. These are your "webpages".
 
-- *MarkDown*:
-  Use the Jekyll theme, probably coupled with GitHub/GitLab pages. Jekyll converts your MarkDown files into HTML, and has some other great features to host a website. See [Using with Jekyll](#using-with-jekyll).
-
-- *HTML*:
-  Use the plugin script. Simply add a couple of lines to your plain HTML webpage and watch it magically transform in your browser! See [Using the plugin](#using-the-plugin).
-
-
-### Using with Jekyll
-*If you plan to use GitHub Pages, see https://pages.github.com for guides.*
-
-Follow these steps to add MarkDown files that use the Primer Spec theme.
-
-1. Add your Markdown files. Note that files named README.md will be ignored by Jekyll when using a custom theme.
-
-2. If it doesn't already exist, create a file `_config.yml` in your site's root directory. Add this content to the file:
-
-    ```yml
-    remote_theme: eecs485staff/primer-spec
-    plugins:
-        - jekyll-remote-theme
-    ```
-
-3. To add the Primer Spec theme to a MarkDown file, add the following at the top of the file:
+2. Add Primer Spec to each "webpage" by inserting the following at the top of the file:
 
     ```yml
     ---
     layout: spec
     ---
     ```
 
-4. (Skip this step if you are using GitHub Pages.) Add the following to your Gemfile:
-
-    ```ruby
-    gem "jekyll-remote-theme"
-    ```
+3. If it doesn't already exist, create a file `_config.yml` in your site's root directory. Add this content to the file:
 
-5. (Optional) To prevent a heading from appearing in the sidebar, add `{: .primer-spec-toc-ignore }` under the heading. For instance:
-
-    ```markdown
-    ### This heading appears in the sidebar
-    
-    Spam spam spam.
-
-    ### This heading does NOT appear in the sidebar.
-    {: .primer-spec-toc-ignore }
-
-    Spam spam spam.
+    ```yml
+    remote_theme: eecs485staff/primer-spec
+    plugins:
+        - jekyll-remote-theme
     ```
 
-6. If you're using GitHub pages, follow the steps in the [USAGE_JEKYLL file](docs/USAGE_JEKYLL.md#Previewing-GitHub-Pages-locally) to preview your website locally.
+4. Deploy your site with GitHub pages!
```

Comment:
Maybe link to the preview locally here?

Reply:
I added a link to the preview right below this line. I've updated the wording in 35dabbfd5fdbbf94a494b88860ba0fe3276fde50 to hopefully be more explicit.